app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -104,10 +107,12 @@ fun ViewProfileScreen(
                   TextButton(
                       modifier = Modifier.testTag("SignOutDialogConfirmButton"),
                       onClick = {
-                        userViewModel.signOut()
-                        Toast.makeText(context, signOutToastText, Toast.LENGTH_SHORT).show()
-                        navigationActions.navigateTo(TopLevelDestinations.AUTH)
-                        signOut = false
+                        userViewModel.signOut {
+                          (context as Activity).runOnUiThread {
```

Comment:
Can you explain to me why you specified that you want the toast to be thrown by the UI thread. Is it not the case already ?